src/neuroconv/datainterfaces/behavior/video/videodatainterface.py

```diff
@@ -437,14 +436,18 @@ def add_to_nwbfile(
                                 pbar.update(1)
                     iterable = video
 
-                # Wrap data for compression
-                wrapped_io_data = H5DataIO(
-                    iterable,
-                    compression=compression,
-                    compression_opts=compression_options,
-                    chunks=chunks,
+            # TODO: remove completely after 03/1/2024
+            if compression is not None or compression_options is not None:
+                warnings.warn(
+                    message=(
+                        "Specifying compression methods and their options at the level of tool functions has been deprecated. "
```

Comment:
```suggestion
                        "Specifying compression methods and their options for this interface has been deprecated. "
```

Copied and pasted from elsewhere? This isn't a tool function, but an interface

Reply:
Yes, I did, thanks for the catch!